apps/nextra/pages/en/build/create-aptos-dapp/templates/nft-minting-dapp.mdx

```diff
@@ -35,8 +35,8 @@ npm run dev
 However, for the app to work as intended, there are several setup steps you will have to do to:
 
 1. Create assets for a collection of NFTs.
-2. Publish the contract to create the collection on-chain.
-3. Actually publish your collection.
+2. Publish the contract and create the collection on-chain.
```

Comment:
Are these separate steps now?

Reply:
i don't think so, we shouldn't change this? @0xmaayan